src/main/java/com/sap/cloudfoundry/client/facade/adapters/RawApplicationLog.java

```diff
@@ -38,8 +40,10 @@ private static String decodeLogPayload(String base64Encoded) {
         return new String(result, StandardCharsets.UTF_8);
     }
 
-    private static Date fromLogTimestamp(String timestamp) {
-        return new Date(TimeUnit.NANOSECONDS.toMillis(Long.parseLong(timestamp)));
+    private static OffsetDateTime fromLogTimestamp(long timestampNanos) {
+        Duration duration = Duration.ofNanos(timestampNanos);
+        Instant instant = Instant.ofEpochSecond(duration.getSeconds(), duration.getNano());
+        return OffsetDateTime.ofInstant(instant, ZoneId.of("UTC"));
```

Comment:
I think we should use ZoneId.systemDefault(), as there might be landscapes that are not in utc

Reply:
The Log Cache returns timestamps in UTC